File: Level_1/problem_1190.java
import java.util.Scanner;

public class Problem_1190 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        
        char op = sc.next().charAt(0);
        
        double[][] matriz = new double[12][12];
        for (int i = 0; i < 12; i++) {
            for (int j = 0; j < 12; j++) {
                matriz[i][j] = sc.nextDouble();
            }
        }
        
        double soma = 0;
        int contagem = 0;
        for (int i = 0; i < 12; i++) {
            for (int j = 0; j < 12; j++) {
                if (j > i && (i + j) > 11) {
                    soma += matriz[i][j];
                    contagem++;
                }
            }
        }
        
        double resultado;
        if (op == 'S') {
            resultado = soma;
        } else {
            resultado = soma / contagem;
        }
        
        System.out.printf("%.1f\n", resultado);

        sc.close();
    }
}

File: Level_1/problem_1190.py
op = input().strip()

matriz = []
for i in range(12):
    linha = []
    for j in range(12):
        valor = float(input().strip())
        linha.append(valor)
    matriz.append(linha)

soma = 0
contagem = 0
for i in range(12):
    for j in range(12):
        if (j > i) and (i + j > 11):
            soma += matriz[i][j]
            contagem += 1

if op == 'S':
    resultado = soma
elif op == 'M':
    resultado = soma / contagem

print(f"{resultado:.1f}")
